modules/gdnative/pluginscript/pluginscript_script.cpp

```diff
@@ -407,6 +408,54 @@ MultiplayerAPI::RPCMode PluginScript::get_rset_mode(const StringName &p_variable
 	}
 }
 
+Variant PluginScript::_new(const Variant **p_args, int p_argcount, Variant::CallError &r_error) {
```

Comment:
This implementation is ignoring the arguments passed to `new`. It will always call the default parameter-less constructor.

Reply:
Thank you for checking. I'd like to fix it, but it will take time, so I will do it tomorrow.

Reply:
I have investigated this problem.

I attempted to call the constructor using the PluginScriptInstance::call() method, but there is currently no way to get the constructor function name of the script in PluginScript.

Ultimately I only wrote comments and warnings.

On the other hand, redefinition of the constructor is prohibited in PythonScript using the PluginScript.
https://github.com/touilleMan/godot-python/blob/master/pythonscript/embedded/godot/hazmat/base.py#L211-L215

I think it is necessary to another discussion to whether constructor arguments are supported in PluginScript.

Reply:
Prohibited redefinition of constructor in Pythonscript is more of a handy hack than a real though design ;-)
On top of that, doing `ResourceLoader.load("res://OtherClass.gd").new()` is currently impossible to do in Pythonscript.
Basically I considered nodes almost always created without parameters (and post-creation configuration can do the trick anyway) so no need to bother with this. That may be also the reason the `new` method is lacking in Pluginscript too (given I developed both at the same time).

Anyway, I understand better you usecase now and I agree with you there is something lacking here :+1:

Reply:
> Basically I considered nodes almost always created without parameters (and post-creation configuration can do the trick anyway) so no need to bother with this.

Yes, I think so too. Basically, if I can initialize a class that inherits a node before adding it to the tree, there is no problem. Therefore, I think that the initialization method is not a critical problem.